connectivity/tests/world.go

```diff
@@ -12,21 +12,23 @@ import (
 
 // PodToWorld sends multiple HTTP(S) requests to one.one.one.one
 // from random client Pods.
-func PodToWorld() check.Scenario {
-	return &podToWorld{}
+func PodToWorld(domain string) check.Scenario {
+	return &podToWorld{domain: domain}
 }
 
 // podToWorld implements a Scenario.
-type podToWorld struct{}
+type podToWorld struct {
+	domain string
+}
 
 func (s *podToWorld) Name() string {
 	return "pod-to-world"
 }
 
 func (s *podToWorld) Run(ctx context.Context, t *check.Test) {
-	http := check.HTTPEndpoint("one-one-one-one-http", "http://one.one.one.one")
-	https := check.HTTPEndpoint("one-one-one-one-https", "https://one.one.one.one")
-	httpsindex := check.HTTPEndpoint("one-one-one-one-https-index", "https://one.one.one.one/index.html")
+	http := check.HTTPEndpoint(s.domain+"-http", "http://"+s.domain)
+	https := check.HTTPEndpoint(s.domain+"-https", "https://"+s.domain)
+	httpsindex := check.HTTPEndpoint(s.domain+"-https-index", fmt.Sprintf("https://%s/index.html", s.domain))
```

Comment:
As noted above, let's get the comain names from `(*check.Test).Context().Params()`:

```suggestion
	domain := t.Context().Params().DomainName
	http := check.HTTPEndpoint(domain+"-http", "http://"+domain)
	https := check.HTTPEndpoint(domain+"-https", "https://"+domain)
	httpsindex := check.HTTPEndpoint(domain+"-https-index", fmt.Sprintf("https://%s/index.html", s.domain))
```

etc.